test/unittests/end_to_end_test.cpp

```diff
@@ -125,8 +126,11 @@ TEST(end_to_end, nested_loops_in_c)
 
     const auto module = parse(bin);
 
+    const auto funcidx = find_exported_function(module, "test");
```

Comment:
I still have a question, why not `func_idx` which I find more readable.

Reply:
Changed.